docs/lib/analytics/fragments/flutter/getting-started/30_initAnalytics.md

```diff
@@ -34,10 +31,9 @@ class MyAmplifyApp extends StatefulWidget {
         AmplifyAuthCognito authPlugin = AmplifyAuthCognito();
         AmplifyAnalyticsPinpoint analyticsPlugin = AmplifyAnalyticsPinpoint();
 
-        amplifyInstance.addPlugin(
-            authPlugins: [authPlugin], analyticsPlugins: [analyticsPlugin]);
+        Amplify.addPlugins([authPlugin, analyticsPlugin]);
```

Comment:
Should we just inline the plugin constructors? That is, remove lines 31-32 and simplify to:
```dart
Amplify.addPlugin(AmplifyAuthCognito());
Amplify.addPlugin(AmplifyAnalyticsPinpoint());
```

Alternately, if you want to show-off the list-style method, we could do:
```dart
Amplify.addPlugins([AmplifyAuthCognito(), AmplifyAnalyticsPinpoint()]);
```
?

Same suggestion throughout the PR

Reply:
Good suggestion. Updated the PR.